### PR TITLE
Dashboard stats

### PR DESCRIPTION
**Pull request by victor-paumier**
- Improve design
- add nationalities stats
- focus stats on current semester, except registration (tooltip to explain)
Capture d’écran 2019-06-14 à 16 01 43

### CODE REVIEW

File: assets/sass/main.scss [hunk 1]
```diff
@@ -113,8 +113,11 @@ h2 {
   }
 }
 
+.tooltip-img {
```
Comment (JeremieSamson) [inline assets/sass/main.scss]: what do you think of this instead ? 

```
.tooltip {
  &-img {
    margin-top: -30px;
  }
  
  &-inner {
    white-space: nowrap;
    max-width: none;
  }
}
```

File: src/Repository/UserRepository.php [hunk 2]
```diff
@@ -311,6 +345,14 @@ private function getFilteredUsers(
             $rolesToExclude = array_merge($rolesToExclude, [UserRoleEnum::ROLE_ADMIN, UserRoleEnum::ROLE_SUPER_ADMIN]);
         }
 
+        if ($forceCurrentSemester) {
+            $qb
+                ->andWhere('uSemester.year = sSemester.year')
+                ->andWhere('uSemester.period = sSemester.period or uSemester.period = :all_year')
+                ->setParameter('all_year', SemesterPeriodEnum::PERIOD_ALL_YEAR)
+                ;
```
Comment (JeremieSamson) [inline src/Repository/UserRepository.php]:
```suggestion
            ;
```

Comment (victor-paumier) [reply src/Repository/UserRepository.php]: done

File: assets/sass/main.scss (hunk 1 above)
Comment (victor-paumier) [reply assets/sass/main.scss]: done

**Pull request merged**